cypress-tests/cypress/support/commands.js

```diff
@@ -1966,10 +1977,13 @@ Cypress.Commands.add(
               .to.have.property("redirect_to_url");
             const nextActionUrl = response.body.next_action.redirect_to_url;
             cy.log(nextActionUrl);
+            for (const key in res_data.body) {
+              expect(res_data.body[key]).to.equal(response.body[key]);
+            }
           } else if (response.body.authentication_type === "no_three_ds") {
-            if (response.body.connector === "fiuu") {
-              expect(response.body.status).to.equal("failed");
-            } 
+            for (const key in res_data.body) {
+              expect(res_data.body[key]).to.equal(response.body[key]);
```

Comment:
expect(res_data.body[key] , [key] ).to.equal(response.body[key]);

Add this way so that we get to know the field name in the response